disco/types/message.py

```diff
@@ -505,6 +541,33 @@ def delete(self):
         """
         return self.client.api.channels_messages_delete(self.channel_id, self.id)
 
+    def toggle_embeds_suppress(self, state=None):
```

Comment:
I'm not a big fan of this function because the default is implicit behavior. I think most use cases will want to be explicit in enabling/disabling embed suppression.

What do you think about `set_embeds_suppressed(bool): Message` instead?